client/lib/abtest/index.js

```diff
@@ -87,6 +88,13 @@ ABTest.prototype.init = function( name ) {
 		throw new Error( 'A default variation is specified for ' + name + ' but it is not part of the variations' );
 	}
 
+	const languageSlugs = map( config( 'languages' ), 'langSlug' );
+	const localeTargets = [ 'not-en', 'any' ].concat( languageSlugs );
+
+	if ( testConfig.localeTargets && ! includes( localeTargets, testConfig.localeTargets ) ) {
+		throw new Error( 'localeTargets can by "any", "not-en" or any single locale slug.' );
```

Comment:
OTOH, from the other POV. After partially absorbing the rest of the code, it feels like the most flexible and intuitive thing would be two arrays: `localTargets` and `localeTargetExcludes` and then just diff values in `if` logic. Is that naive? Is there a restricting factor my drive-by view is missing that limits us to single values?